Guided Investigation - MDATP Webshell Alerts.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<p>This will fail if you specify a workspace_name that you don't have set so it may be good to update the error message to state that you don't have the specified workspace configured in order to be more accurate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Instead of changing this I have limited to only allow the user to select a workspace that is in their config in addition to moving the config error checking code around this new dropdown.</p>